### DISCUSSION TITLE
Allow clones of board games?

### DISCUSSION THREAD

**Discussion by cxong** [opengaming/osgameclones#2789] [Q&A] [answered]
I feel like we should draw a line somewhere, should we allow clones of board games?

e.g. #870 #871

**Comment by OvermindDL1** [comment]
I'd personally say if they are computer game clones of board games then it's fine, but if they are physical clones of other physical games, then probable not?  Though the name `osgameclones` is generic enough that it could feasibly still include those?

**Comment by pehsa** [comment, marked as answer]
I think if the board game has/had an official video game, than okay. For example Monopoly or Risk. In the original section we need a plus board game platform.

The mainstream board or card games 90% has/had an official video game. The TEG board game already a clone of Risk so I don't think we need to list all of them.

Or we create a new section only for board and card games. It will not make the main section tremendous and separate well. Card games, like Patience/Solitaire games has so many video game clones. If we collect all clones, than we will have a list of 20-30-40 title.
